src/CSS/book.css

```diff
@@ -4,7 +4,7 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
-  border: 0.1rem solid black;
+  border: 0.1rem solid #888;
```

Comment:
- [ ] Please use the same border color used in the zephin design, you can also add border-radius so your design looks exactly like the Figma design 😉